test/pdfs/.gitignore

```diff
@@ -355,6 +359,7 @@
 !openoffice.pdf
 !issue7014.pdf
 !issue8187.pdf
+!StrikeOut.without.appearance.stream.pdf
```

Comment:
Let's call this file `annotation-strikeout-without-appearance.pdf` and put it below the existing `annotation-strikeout.pdf` to group them a bit more and to have a similar naming scheme. Similar for the other three files.